test/power_divergence.jl

```diff
@@ -158,6 +174,8 @@ confint(m, method = :sison_glaz, tail=:right)
 @test_throws ArgumentError confint(m, method=:FOO)
 @test_throws ArgumentError confint(m, tail=:fox)
 
+@test confint(m, method = :sison_glaz) == confint(m, method = :auto) == confint(m)
```

Comment:
Also test a case where the other method is used?

Reply:
It is tested by the other set of tests above.